website/public/benchmarks/criterion-json-to-html.mjs

```diff
@@ -69,9 +69,15 @@ export function makeBenchmarkHTML(seriess) {
       )}
 
       <div class="x-axis-label">response time (lower is better)</div>
+      <!-- 
+        title attribute must be broken into new line
+        because the html spec treats &#013; as a parse error, which
+        gives us validation errors.
+      -->
       <label
         class="log-scale javascript-only"
-        title="checked: logarithmic X axis (emphasizing ratios).&#013;unchecked: linear X axis (emphasizing time)."
+        title="checked: logarithmic X axis (emphasizing ratios).
+        unchecked: linear X axis (emphasizing time)."
```

Comment:
Blocking: This adds a bunch of spaces before the word 'unchecked':
<img width="302" alt="Screen Shot 2021-06-04 at 00 14 53" src="https://user-images.githubusercontent.com/48666/120761376-f999aa00-c4c9-11eb-83a2-a49193e04d11.png">

It's supposed to be flush with the left side:
<img width="290" alt="Screen Shot 2021-06-04 at 00 15 09" src="https://user-images.githubusercontent.com/48666/120761379-facad700-c4c9-11eb-8462-0c0730c4d034.png">

Reply:
Would you like me to change it back to how it was initially? If so, we will still get the following validation error: 
```
A numeric character reference expanded to carriage return
```

I was reading about the [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#multiline_titles) attribute and removing the extra spacing after the line break seems to give the right behavior.

Reply:
Your latest version looks good to me.